docs/specification/draft/basic/enterprise.mdx

```diff
@@ -0,0 +1,297 @@
+---
+title: Enterprise Authorization Profile
+---
+
+<Info>**Protocol Revision**: draft</Info>
+
+## 1. Introduction
+
+### 1.1 Purpose and Scope
+
+This document defines an application of the "[Identity Assertion Authorization Grant](https://datatracker.ietf.org/doc/draft-parecki-oauth-identity-assertion-authz-grant/)" for use within enterprise deployments of the Model Context Protocol (MCP). It specifies how an MCP Client can obtain an access token from an MCP Server's Authorization Server by presenting an identity assertion, and how an MCP Server's Authorization Server validates and utilizes this identity assertion before issuing an access token. This profile is designed to facilitate secure and interoperable authorization within enterprise environments, leveraging the organization's existing identity infrastructure.
+
+Enterprise-ready authorization for MCP Clients and Servers has two key benefits:
+
+- For end users, this removes the need to manually connect and authorize the MCP Client to each MCP Server for use within the organization.
+- For enterprise admins, this enables visibility and control over which MCP Servers are able to be used within the organization.
+
+### 1.2 Roles and Terminology
+
+- **Resource Application**: In the context of this profile, the Resource Application is the MCP Server.
+- **Identity Provider (IdP)**: An entity that creates, maintains, and manages identity information, provides authentication services, and is trusted by a set of applications in an organization's app ecosystem.
+- **Identity Assertion**: A security token (e.g., ID Token, SAML Assertion) that contains information about an authenticated user and is issued by an Identity Provider trusted by the Authorization Server.
+- **Authorization Server**: This profile uses the term "Authorization Server" to refer to the authorization server that issues tokens to be used at the MCP server.
+- **Client**: In the context of this profile, the Client is the MCP Client.
+- **ID Token**: A security token that contains claims about the authentication of a user when using a client, as defined in [OpenID Connect](https://openid.net/specs/openid-connect-core-1_0.html#IDToken).
+- **Subject Token**: A security token that represents the identity of the user, such as an ID Token.
+
```

Comment:
Should we add a definition here was was JAG stands for (JWT Authorization Grant)?